lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -388,9 +388,9 @@ struct TypeLoweringVisitor : public FIRRTLVisitor<TypeLoweringVisitor, bool> {
   bool visitExpr(SubaccessOp op);
   bool visitExpr(VectorCreateOp op);
   bool visitExpr(BundleCreateOp op);
-  bool visitExpr(ElementwiseAndPrimOp op);
-  bool visitExpr(ElementwiseOrPrimOp op);
-  bool visitExpr(ElementwiseXorPrimOp op);
+  bool visitExpr(AndVecOp op);
+  bool visitExpr(OrVecOp op);
+  bool visitExpr(XorVecOp op);
```

Comment:
Similarly, the other (new) operations probably should be handled here too?

For this sort of reason (+tests) it might/often helps to split this sort of thing into multiple PR's.